internal/otelcollector/config/trace/gateway/config.go

```diff
@@ -13,8 +13,9 @@ type Config struct {
 }
 
 type Receivers struct {
-	OpenCensus config.Endpoint     `yaml:"opencensus"`
-	OTLP       config.OTLPReceiver `yaml:"otlp"`
+	OpenCensus        config.Endpoint     `yaml:"opencensus"`
+	OTLP              config.OTLPReceiver `yaml:"otlp"`
+	OTLPTraceReceiver config.OTLPReceiver `yaml:"otlp/trace-receiver"`
```

Comment:
Let's find a better name. "trace-receiver" could also be used for the two other receivers.